code/controllers/subsystem/persistence.dm

```diff
@@ -0,0 +1,60 @@
+var/datum/subsystem/persistence/SSpersistence
+
+/datum/subsystem/persistence
+	name = "Persistence"
+	init_order = -100
+	flags = SS_NO_FIRE
+	var/savefile/secret_satchels
+	var/list/new_secret_satchels = list() //these are objects
+	var/list/old_secret_satchels = list() //these are associate
+
+/datum/subsystem/persistence/New()
+	NEW_SS_GLOBAL(SSpersistence)
+
+/datum/subsystem/persistence/Initialize()
+	PlaceSecretSatchel()
+	..()
+
+/datum/subsystem/persistence/proc/CollectData()
+	CollectSecretSatchels()
+
+/datum/subsystem/persistence/proc/PlaceSecretSatchel()
+	var/list/satchels = list()
+	var/list/invalid_satchels = list()
+	secret_satchels = new /savefile("data/npc_saves/SecretSatchels.sav")
+	while(!secret_satchels.eof)
+		var/list/potential_satchel = list()
+		secret_satchels >> potential_satchel
+		if(potential_satchel["map"] != MAP_NAME)
+			invalid_satchels += potential_satchel
+			continue
+		satchels += potential_satchel
+	if(satchels.len < 1)
+		return
+	var/list/chosen_satchel = pick_n_take(satchels)
+	old_secret_satchels = satchels + invalid_satchels
+	secret_satchels << old_secret_satchels
+	var/path = text2path(chosen_satchel["object"])
+	var/obj/item/weapon/storage/backpack/satchel/flat/secret/F = new(chosen_satchel["loc"])
+	new path(F) //runtimes intentionally if satchel item has had its path changed/removed
```

Comment:
Is there a more elegant way to check to see if a path is valid that doesn't provoke a runtime? I know about `try` but the way it works doesn't really seem like it'd prevent the runtime.

Comment is mostly a meme placeholder while I look for a better solution